packages/libra-rpc-graphql/package.json

```diff
@@ -17,7 +17,7 @@
     "preserver": "yarn run introspectify:server",
     "server": "ts-node scripts/server.ts",
     "prestart": "yarn run introspectify:server",
-    "start": "ts-node-dev --respawn --transpileOnly scripts/server.ts"
+    "start": "ts-node-dev --respawn --transpile-only scripts/server.ts"
```

Comment:
this has [always been `--transpile-only`](https://github.com/whitecolor/ts-node-dev#usage), it probably wasn't working before